specs/scenarios/trust-store-and-trust-policy.md

```diff
@@ -0,0 +1,212 @@
+This document is intented to describe how the trust store and the trust policies will be used in real-live applications. 
+The document will concentrate on container-based applications but the concepts can be extended to any other type of application. 
+
+## Related Links
+The following documents are related to the scenarios described in this document. Reading those will provide additional context:
+- [Notary v2 Signing Scenarios](https://github.com/notaryproject/notaryproject/blob/main/scenarios.md#scenario-0-build-publish-consume-enforce-policy-deploy)
+- [Trust Store and Trust Policy Specification](https://github.com/notaryproject/notaryproject/blob/main/specs/trust-store-trust-policy.md)
+- [Key Management Requirements](https://github.com/notaryproject/notaryproject/blob/main/keymanagementrequirements.md)
+
+## Background Information
+For the purpose of the scenarios clarity, we will start with a description of a modern containerized application and how it is deployed and used by customers. This section will also introduce some terminology that will be used later on in the document.
+
+Many modern applications are deployed on a *Kubernetes cluster*. In many cases the cluster is used in a multi-tenant scenarios and it is managed by either the cloud provider or a central team in the organization. Also, for the purposes of the Kubernetes' operation, containers used to manage the cluster are deployed in the `kube-system` namespace. The *Kubernetes cluster* is considered the base or *platform layer* for our scenarios.
+
+Cloud vendors or internal organizations deploy *services* on top of Kubernetes. Those services can provide direct benefits to end users or can be used by other service teams to extend the capabilities of the platform. One example of such services are Batch service that can be used by customers to run a large scale jobs on top of Kubernetes. Each one of those services can be deployed in a separate namespace to provide seggregation. We will call this layer the *service layer* for our scenarios.
+
+Customers who leverage the services, can deploy their own workloads in the form of containers. In the above example with Batch, customers may build a map-reduce application that runs the mapping and reducing tasks as containers and deploy it on top of the Batch service. This is the *application layer* for our scenarios. 
+
+Here is how this stack looks like:
+
+| layer | description | example |
+| ----- | ----- | ----- |
+| application | Customer application deployed in the form of containers using the services provided in the previous layer | Map-Reduce Application|
+| service | Containerized services deployed on the platform | Batch Service |
+| platform | Infrastructure management platform allowing elastic expansion of the workloads | Kubernetes | 
+
+To further clarify the distinction between the service workloads and the application workaloads, we can use the following diagram.
+
+![](../../media/trust-store-scenarios-app-stack.svg)
```

Comment:
Is [Kubernetes Control Plane](https://kubernetes.io/docs/concepts/overview/components/#control-plane-components) more commonly-used than *Kubernetes Management Plane*?

Reply:
I don't know :) I am used to "control plane"